Libraries/LibVT/Range.h

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
```

Comment:
I didn't write this :)

Reply:
Ah i assumed all files in serenity had to have that disclamer, should i just remove the copyright statement or replace it with my name?

Reply:
Two options:
1) Put your own name
2) Put "the SerenityOS developers"

And please use 2020 as the copyright year instead of 2018-2020 :)